go/vt/sqlparser/sql.y

```diff
@@ -7854,6 +7893,7 @@ non_reserved_keyword:
 | OJ
 | OLD
 | ONLY
+| OPEN
```

Comment:
Add new non-reserved keywords to the list `correctlyDoParse` in parse_test.go

Reply:
They're already there!
https://github.com/dolthub/vitess/blob/main/go/vt/sqlparser/parse_test.go#L5780
https://github.com/dolthub/vitess/blob/main/go/vt/sqlparser/parse_test.go#L5991
James must have added them a while ago even though they weren't used keywords yet.